activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+*   Add `unfreeze_time` alias method to `travel_back` in `ActiveSupport::Testing::TimeHelpers`.
+
+    The method unstubs `Time.now` and provides a method with a verb symmetry
```

Comment:
Better to keep this brief.

Since the first line says `unfreeze_time` is an alias of `travel_back`, the functionility is already known (as far as what to include in the CHANGELOG is concerned). You can include the rationale though, something like

> The alias is provided for symmetry with `freeze_time`.

would be enough.

Reply:
I can understand that readers could look up the functionality. This more succinct explanation is now added.